### DISCUSSION TITLE
Order of suggestions in autocomplete

### DISCUSSION THREAD

**Discussion by Wyall** [James-Yu/LaTeX-Workshop#3643] [Q&A] [answered]
Hi everybody,

I am using the siunitx package and its ```\qty{}{}``` command a lot. However, when I start typing ```\qty``` the autocomplete always suggests ```\qtylist```, ```\qtyproduct``` and ```\qtyrange``` from the same package first, which is annoying as they are used less often. Is there a way to change this behaviour?

Edit: Nevermind, now ```\qty``` is highlighted by default, even as the fourth entry.

**Comment by jlelong** [comment, marked as answer]
VS Code learns from your previous completions when `editor.suggestSelection` is set to `recentlyUsed` or `recentlyUsedbyPrefix`.
